mavros_msgs/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ add_message_files(
   DebugValue.msg
   ExtendedState.msg
   FileEntry.msg
+  Float32ArrayStamped.msg
```

Comment:
I think you may use some std_msgs? multiarray?

Reply:
MultiArray is not stamped. But if we are to create a new msg @pavloblindnology, you can basically name it `WheelOdomStamped.msg`